commons/src/test/java/org/sonarsource/analyzer/commons/RuleMetadataLoaderTest.java

```diff
@@ -233,6 +235,34 @@ public void getStringArray() throws Exception {
     assertThat(RuleMetadataLoader.getStringArray(map, "key")).containsExactly("x", "y");
   }
 
+  @Test
+  public void getScopeIfPresent() {
+    RuleScope scope;
+
+    assertThat(RuleMetadataLoader.getScopeIfPresent(Collections.emptyMap(), "scope")).isNotPresent();
+    assertThat(RuleMetadataLoader.getScopeIfPresent(Collections.singletonMap("scope", null), "scope")).isNotPresent();
```

Comment:
Use static import for `singletonMap` and `emptyMap` to make code a bit more succint

Reply:
Indeed I saw that too. The PR doesn't show that the same methods are used many times in the same file. There's undoubtedly an opportunity to refactor the class, for instance, to use the shorter form `List.of()`. I may do this as part of a separate PR, maybe? WDYT?

Reply:
we would need to change the `pom.xml` to use Java 11, otherwise `List.of` is not available. We could do this, but perhaps later.